madgraph.sh

```diff
@@ -0,0 +1,28 @@
+package: madgraph
+version: "%(tag_basename)s"
+tag: "v2.8.1"
+source: https://github.com/mfasDa/MadGraph5
+---
+#!/bin/bash -e
+
+rsync -a --exclude='**/.git' --delete --delete-excluded $SOURCEDIR/ $INSTALLROOT/
```

Comment:
Does this mean that this is a binary only distribution?

Reply:
Yes, it is a pure binary (python). However for NLO process calculations i.e. within Herwig it produces files which needs to be compiled on-the-fly. Sherpa does something similar. This is different with respect to openloops where the process libraries are static and can be provided with the package. All this is steered from within Herwig, which only needs to know where to find the binary.

For process calculation on the grid as far as I see in local tests the most efficient is to build the processes and run the integration locally - because this is time consuming (waste of time if we repeat it for each job), can be parallelized on clusters, and can be reused from all jobs - and copy the process library and the grids to the grid (working dir), and just run event generation there.

Reply:
I want to add that "binaries" (mg5_aMC) take care about the PYTHONPATH, so this can be skipped.